README.md

```diff
@@ -7,6 +7,7 @@ Use when a library uses `slog` but your application uses `tracing`.
 
 Heavily inspired by [tracing-log](https://github.com/tokio-rs/tracing/tree/60c60bef62972e447414a748a95b31ff9027165b/tracing-log).
 
-Specifically, the emitted log entries include the custom fields `slog.target`, `slog.module_path`, `slog.file`, `slog.line`, and `slog.column` with corresponding values from the slog call site.
+Specifically, the emitted log entries include the custom fields `slog.target`, `slog.module_path`, `slog.file`, `slog.line`, and `slog.column` with corresponding values from the slog call site. A field `slog.fields` will contain a comma-separated list of the possible [key value pairs](https://docs.rs/slog/latest/slog/trait.KV.html) supported in `slog` logging macros if the `kv` feature is enabled.
```

Comment:
Would it work for you to call the field `kv` instead of `fields`? That's the name in slog, and `fields` seems a bit overloaded here.

Reply:
Sure, sounds good! will update everywhere